src/Constants.ts

```diff
@@ -8,6 +8,11 @@ export const CONFIG_FILE = 'Config.json';
 export const PMD_CATALOG_FILE = 'PmdCatalog.json';
 export const INTERNAL_ERROR_CODE = 500;
 
+export const PILOT_AVAILABILITY_BANNER= `As of April 27, 2022, v3.x of the scanner is available as an open pilot.
+         Please consider updating to v3.x, by running the following commands:
+         > sfdx plugins:uninstall @salesforce/sfdx-scanner
+         > sfdx plugins:install @salesforce/sfdx-scanner@latest-pilot`;
```

Comment:
NOTE: The indenting on lines 2-4 is so that it's all evenly indented when you take into account the "WARNING: " prefix on line 1.

Reply:
**Edit:**

To update to version 3.x and try out our pilot features, run these commands:

(Small CCX writing tip: don't use please. =) It seems polite to do so, yes, but we only use please under the rarest, you-really-really-must-do-this terms)